src/helpers/snapshot.ts

```diff
@@ -49,50 +49,38 @@ export async function getResults(space, proposal, votes, blockNumber) {
     console.timeEnd('getProposal.scores');
     console.log('Scores', scores);
 
-    const authorProfile = profiles[proposal.author];
-    voters.forEach(address => {
-      votes[address].profile = profiles[address];
-    });
-    proposal.profile = authorProfile;
+    proposal.profile = profiles[proposal.author];
+    votes.map(vote => (vote.profile = profiles[vote.voter]));
 
-    votes = Object.fromEntries(
-      Object.entries(votes)
-        .map((vote: any) => {
-          vote[1].scores = strategies.map(
-            (strategy, i) => scores[i][vote[1].address] || 0
-          );
-          vote[1].balance = vote[1].scores.reduce((a, b: any) => a + b, 0);
-          return vote;
-        })
-        .sort((a, b) => b[1].balance - a[1].balance)
-        .filter(vote => vote[1].balance > 0)
-    );
+    votes.map((vote: any) => {
+      vote.scores = strategies.map((strategy, i) => scores[i][vote.voter] || 0);
+      vote.balance = vote.scores.reduce((a, b: any) => a + b, 0);
+      return vote;
+    });
+    votes
+      .sort((a, b) => b.balance - a.balance)
+      .filter(vote => vote.balance > 0);
```

Comment:
For some reason if you go here:
https://snapshot.org/#/ybaby.eth/proposal/QmSMyYeKrRpnA7Xn56o2NtbCUzxmhzCupL7LxMA1reXxq4
You'll see 1922 votes, but if you go here: 
https://bafybeibiegge3dzfmnz6esriceobmyz5fqgbif37woivpmmvwbcgmcggdy.on.fleek.co/#/ybaby.eth/proposal/QmSMyYeKrRpnA7Xn56o2NtbCUzxmhzCupL7LxMA1reXxq4
You see 1923 votes. The extra vote seem to come from this address 0x092a...Ff04 but this address seem to have 0 voting power show it shouldn't really appear. Maybe the filtering `vote.balance > 0` is not working, can you troubleshot this?

Reply:
Should work now